Update docs for Forms 13 (#6152)
umbraco/UmbracoDocs: 13/umbraco-forms/developer/extending/setting-types.md

| Textfield                 | Used a single-line textbox for entry                              |                                                 |
| TextfieldNoAutocomplete   | Used a single-line textbox for entry (with autocomplete disabled) |                                                 |
| TextWithFieldPicker       | Used a single-line textbox/form field list for entry              | Not used in core types                          |
| MultipleTextString        | Used multiple textboxes for multiple entries                      | Not used in core types                          |

All of the above setting types are used in one or more field, prevalue source and workflow types available with Umbraco Forms. For the less common ones, a usage has been indicated in the table.

> The two exceptions are "TextWithFieldPicker" and "MultipleTextString". These two we don't use within the package, but we make it available for developers to use when creating their own types.

Reviewer: ```suggestion
The two exceptions are "TextWithFieldPicker" and "MultipleTextString". We do not use these two within the package, but we make them available for developers to use when creating their own types.
```

> "TextWithFieldPicker" offers the option of text field entry or the selection of a field from the form. This can be useful in workflows where you need to reference the value of a specific field.

Reviewer: ```suggestion
"TextWithFieldPicker" offers the option of text field entry or selection of a field from the form. This can be useful in workflows where you need to reference the value of a specific field.
```
